src/Omega_h_mesh.hpp

```diff
@@ -125,26 +125,50 @@ class Mesh {
   Graph ask_star(Int dim);
   Graph ask_dual();
 
-  Adj ask_revClass (Int edim);
+/* ask_revClass (Int edim, LOs g_ids): takes input of entity dimension
```

Comment:
Please describe the relationship between the model and mesh entity dimension and the 'edim' input argument.